phpstan-baseline.neon

```diff
@@ -0,0 +1,15 @@
+
+
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Access to undefined constant Stripe\\\\ApiResource\\:\\:OBJECT_NAME\\.$#"
+			count: 1
+			path: lib/ApiResource.php
+
+		-
+			message: "#^Access to undefined constant Stripe\\\\SingletonApiResource\\:\\:OBJECT_NAME\\.$#"
+			count: 1
+			path: lib/SingletonApiResource.php
```

Comment:
Note that these two errors are "real" issues: there is no guarantee that the `OBJECT_NAME` constant will exist in these methods, and there is no mechanism to force a derived class to declare a specific static constant. We should probably replace the `OBJECT_NAME` constant with a `getObjectName()` method that would be declared as `abstract` in `ApiResource`.

This would be a non-trivial refactor though and the current code has been working for a long time, so let's not leave it as is for now.